tests/BenchmarkDotNet.Tests/Portability/Cpu/CpuInfoFormatterTests.cs

```diff
@@ -28,7 +24,9 @@ public void FormatTest()
                 captions.AppendLine(CpuInfoFormatter.Format(mockCpuInfo));
             }
 
-            Approvals.Verify(captions.ToString());
+            var settings = new VerifySettings();
+            settings.UseDirectory("ApprovedFiles");
```

Comment:
We use the same name for the directory with verified files across all the tests. It would be nice to avoid duplicating of the "ApprovedFiles" string in multiple places.